bin/consequence_mapping/severity_ranking.txt

```diff
@@ -0,0 +1,36 @@
+transcript_ablation
+splice_acceptor_variant
+splice_donor_variant
+stop_gained
+frameshift_variant
+stop_lost
+start_lost
+transcript_amplification
+inframe_insertion
+inframe_deletion
+missense_variant
+protein_altering_variant
+splice_region_variant
+incomplete_terminal_codon_variant
+start_retained_variant
+stop_retained_variant
+synonymous_variant
+coding_sequence_variant
+mature_miRNA_variant
+5_prime_UTR_variant
+3_prime_UTR_variant
+non_coding_transcript_exon_variant
+intron_variant
+NMD_transcript_variant
+non_coding_transcript_variant
+upstream_gene_variant
+downstream_gene_variant
+TFBS_ablation
+TFBS_amplification
+TF_binding_site_variant
+regulatory_region_ablation
+regulatory_region_amplification
+feature_elongation
+regulatory_region_variant
+feature_truncation
+intergenic_variant
```

Comment:
Have you checked with Ensembl VEP devs if this list is something that could be retrieved with a web service (so that we don't need to have this hard-coded text file)?

Reply:
There is an endpoint to retrieve all consequence types: https://rest.ensembl.org/documentation/info/variation_consequence_types

Unfortunately, it returns the consequence types in arbitrary order and does not specify the severity rankings.